frontend_lib/src/component/CardPopup/CardPopupCreateContent.jsx

```diff
@@ -1,98 +1,108 @@
-        <CardPopup
-          customClass='popupCreateContent'
-          customColor={props.customColor}
-          onClose={props.onClose}
-          customStyle={props.customStyle}
-          faIcon={props.faIcon}
-          label={props.label}
-        >
-          <div className='createcontent'>
-            <form className='createcontent__form' data-cy='createcontent__form'>
-              {props.children
-                ? props.children
-                : (
-                  <input
-                    type='text'
-                    className='createcontent__form__input'
-                    data-cy='createcontent__form__input'
-                    placeholder={props.inputPlaceholder}
-                    value={props.contentName}
-                    onChange={props.onChangeContentName}
-                    onKeyDown={this.handleInputKeyDown}
-                    autoFocus
-                  />
-                )}
-              <div className='createcontent__form__button'>
-                <button
-                  type='button' // do neither remove this nor set it to 'submit' otherwise clicking the btn will submit the form and reload the page
-                  className='createcontent__form__button btn highlightBtn'
-                  data-cy='popup__createcontent__form__button'
-                  onClick={props.onValidate}
-                  style={{
-                    backgroundColor: props.customColor,
-                    color: '#fdfdfd',
-                    borderColor: props.customColor,
-                    ':hover': {
-                      backgroundColor: color(props.customColor).darken(0.15).hex()
-                    }
-                  }}
-                  disabled={!props.contentName || props.contentName.length === 0}
-                >
-                  {props.btnValidateLabel}
-                </button>
-              </div>
-            </form>
-          </div>
-        </CardPopup>
-      </div>
-    )
-  }
+  return (
+    <div className='popupCreateContentOverlay'>
+      <CardPopup
+        customClass='popupCreateContent'
+        customColor={props.customColor}
+        onClose={props.onClose}
+        customStyle={props.customStyle}
+        faIcon={props.faIcon}
+        label={props.label}
+      >
+        <div className='createcontent'>
+          <form className='createcontent__form' data-cy='createcontent__form'>
+            {props.children
+              ? props.children
+              : (
+                <input
+                  type='text'
+                  className='createcontent__form__input'
+                  data-cy='createcontent__form__input'
+                  placeholder={props.inputPlaceholder}
+                  value={props.contentName}
+                  onChange={props.onChangeContentName}
+                  onKeyDown={handleInputKeyDown}
+                  autoFocus
+                />
+              )}
+            <span>À partir d'un model</span>
```

Comment:
```suggestion
            <span>{props.t('Starting from a template')</span>
```

Reply:
It would be good to check if the template list is empty and have an "allowTemplates" variable so that we can hide this option for threads.